rpc/src/crab.rs

```diff
@@ -242,39 +225,68 @@ where
 		// Whether to format the `peer_count` response as Hex (default) or not.
 		true,
 	)));
-	io.extend_with(Web3ApiServer::to_delegate(Web3Api::new(client)));
-	// --- dvm --->
+	io.extend_with(Web3ApiServer::to_delegate(Web3Api::new(client.clone())));
 
-	Ok(io)
-}
+	if ethapi_debug_targets
```

Comment:
Please comment the `TraceApiServer` and `DebugApiServer` here.

Reply:
I prefer to disable them in the cargo feature.

Reply:
We can control it via `evm-tracing`.

Reply:
So we can keep the code here. Without comment it.